### PR TITLE
Subscription center update

### PR DESCRIPTION
**Pull request by lkpttn**
### What's this PR do?

Creates a new subscription center interface to allow users to toggle individual subscriptions on and off. Reads from the new northstar resource to POST or DELETE individual descriptions.

### How should this be reviewed?

Run it yourself, check the styles and your subscription status.

### Relevant tickets

References [Pivotal #171544380](https://www.pivotaltracker.com/story/show/171544380).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [x] Documentation added for new features/changed endpoints.
- [x] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.

Screen Shot 2020-03-13 at 1 04 03 PM
Screen Shot 2020-03-13 at 13 15 43
Screen Shot 2020-03-13 at 13 15 59

### CODE REVIEW

File: resources/assets/components/pages/AccountPage/EmailSubscriptionItem.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        >
+          {topics.includes(identifier) ? 'Unsubscribe' : 'Subscribe'}
+        </button>
+      </div>
+    </div>
+  );
+};
+
+EmailSubscriptionItem.propTypes = {
+  identifier: PropTypes.string.isRequired,
```
Comment (katiecrane) [inline resources/assets/components/pages/AccountPage/EmailSubscriptionItem.js]: I was a little confused about this, do you prefer `identifier` or `topic`?

**Comment by katiecrane** [comment]
> It's not done yet!
This is good for production though yeah?

**Comment by lkpttn** [comment]
> > It's not done yet!
> 
> This is good for production though yeah?

Oh yeah I forgot to update the description rip.

File: resources/assets/components/pages/AccountPage/EmailSubscriptionItem.js (hunk 1 above)
Comment (lkpttn) [reply resources/assets/components/pages/AccountPage/EmailSubscriptionItem.js]: This is a good point, I've changed the prop to topic

**Pull request merged**